plugin/package.json

```diff
@@ -2,11 +2,12 @@
   "name": "workflow-ui",
   "version": "1.0.0",
   "description": "Jenkins Pipeline UI",
+  "private": true,
```

Comment:
See [the documentation](https://docs.npmjs.com/cli/v9/configuring-npm/package-json#private). This is to prevent an accidental publish of our code.